mappings/net/minecraft/data/client/model/BlockStateModelGenerator.mapping

```diff
@@ -1,12 +1,422 @@
+	METHOD method_25545 registerFlowerPotPlant (Lnet/minecraft/class_2248;Lnet/minecraft/class_2248;Lnet/minecraft/class_4910$class_4913;)V
+		ARG 1 plantBlock
+		ARG 2 flowerPotBlock
+		ARG 3 tintType
+	METHOD method_25546 registerCubeWithCustomTexture (Lnet/minecraft/class_2248;Lnet/minecraft/class_2248;Ljava/util/function/BiFunction;)V
+		ARG 1 block
+		ARG 2 otherTextureSource
+		ARG 3 textureFactory
+	METHOD method_25547 registerCrop (Lnet/minecraft/class_2248;Lnet/minecraft/class_2769;[I)V
+		ARG 1 crop
+		ARG 2 ageProperty
+		ARG 3 ageTextureIndices
+	METHOD method_25548 registerTintableCross (Lnet/minecraft/class_2248;Lnet/minecraft/class_4910$class_4913;)V
+		ARG 1 block
+		ARG 2 tintType
+	METHOD method_25549 registerTintableCross (Lnet/minecraft/class_2248;Lnet/minecraft/class_4910$class_4913;Lnet/minecraft/class_4944;)V
+		ARG 1 block
+		ARG 2 tintType
+		ARG 3 texture
+	METHOD method_25550 registerNorthDefaultHorizontalRotatable (Lnet/minecraft/class_2248;Lnet/minecraft/class_4944;)V
+		ARG 1 block
+		ARG 2 texture
+	METHOD method_25551 registerSingleton (Lnet/minecraft/class_2248;Lnet/minecraft/class_4944;Lnet/minecraft/class_4942;)V
+		ARG 1 block
+		ARG 2 texture
+		ARG 3 model
+	METHOD method_25553 registerAxisRotated (Lnet/minecraft/class_2248;Lnet/minecraft/class_4946$class_4947;)V
+		ARG 1 block
+		ARG 2 modelFactory
+	METHOD method_25554 registerAxisRotated (Lnet/minecraft/class_2248;Lnet/minecraft/class_4946$class_4947;Lnet/minecraft/class_4946$class_4947;)V
+		ARG 1 block
+		ARG 2 verticalModelFactory
+		ARG 3 horizontalModelFactory
+	METHOD method_25555 registerDecorative (Lnet/minecraft/class_2248;Lnet/minecraft/class_4946;)Lnet/minecraft/class_4910$class_4912;
+		ARG 1 block
+		ARG 2 model
+	METHOD method_25556 registerItemModel (Lnet/minecraft/class_2248;Ljava/lang/String;)V
+		ARG 1 block
+		ARG 2 textureSuffix
+	METHOD method_25557 createSubModel (Lnet/minecraft/class_2248;Ljava/lang/String;Lnet/minecraft/class_4942;Ljava/util/function/Function;)Lnet/minecraft/class_2960;
+		ARG 1 block
+		ARG 2 suffix
+		ARG 3 model
+		ARG 4 textureFactory
+	METHOD method_25558 registerBeehive (Lnet/minecraft/class_2248;Ljava/util/function/Function;)V
+		ARG 1 beehive
+		ARG 2 textureGetter
+	METHOD method_25560 registerTopSoil (Lnet/minecraft/class_2248;Lnet/minecraft/class_2960;Lnet/minecraft/class_4935;)V
+		ARG 1 topSoil
+		ARG 2 modelId
+		ARG 3 snowyVariant
+	METHOD method_25561 registerPiston (Lnet/minecraft/class_2248;Lnet/minecraft/class_2960;Lnet/minecraft/class_4944;)V
+		ARG 1 piston
+		ARG 2 extendedModelId
+		ARG 3 texture
+	METHOD method_25565 createBooleanPropertyBasedModelMap (Lnet/minecraft/class_2746;Lnet/minecraft/class_2960;Lnet/minecraft/class_2960;)Lnet/minecraft/class_4926;
+		ARG 0 property
+		ARG 1 trueModel
+		ARG 2 falseModel
+	METHOD method_25566 createValueMarkedVariantMap (Lnet/minecraft/class_2769;Ljava/lang/Comparable;Lnet/minecraft/class_2960;Lnet/minecraft/class_2960;)Lnet/minecraft/class_4926;
```

Comment:
What does "value marked" mean?

Reply:
The value serves as a fence; a value in the property higher than to equal to this mark (I should probably rename to fence) value would give the variant map (a variant map is a subset of block state properties to a subset of model variant parameters (model, rotations, etc), such as powered -> model, facing -> x/y, etc) a model that is `higherModel`, while the other values, less than the mark, would be associated their model be this `lowerModel`.